includes/chart.js

```diff
@@ -336,6 +336,43 @@ function updateEdges(rowID, edgesGroup) {
         // Append the line to the edges group
         edgesGroup.appendChild(line);
 
+        // Add deontic text between second and third nodes
+        if (link.source === 2 && link.target === 3) {
+
+            // Find the row object that corresponds to the current rowID
+            let rowObj = rowValues.find(row => row[0] === rowID);
+
+            // Check if 'deontic' attribute is defined in the array
+            if (rowObj && rowAttributes.indexOf('deontic') !== -1) {
+
+                let deonticIndex = rowAttributes.indexOf('deontic'); // Get the index of the 'deontic'
+                let deonticValue = rowObj[deonticIndex]; // Get the deontic value
+
+                if (deonticValue) {
+
+                    let textElement = document.createElementNS("http://www.w3.org/2000/svg", "text"); // Create a new SVG text element
+
+                    // Calculate the midpoint of the line connecting source and target nodes
+                    let midX = (sourceX + targetX) / 2;
+                    let midY = (sourceY + targetY) / 2 - 6; // Slightly adjust the y-position to avoid overlap
+
+                    // Set attributes for the text element
+                    textElement.setAttribute("x", midX);  // Set the x-coordinate to the midpoint
+                    textElement.setAttribute("y", midY);  // Set the y-coordinate to the midpoint
+                    textElement.setAttribute("fill", "gray");  // Set the text color to black
+                    textElement.setAttribute("font-size", "12px");  // Set the font size
+                    textElement.setAttribute("font-weight", "bold");  // Set the font weight to bold
+                    textElement.setAttribute("text-anchor", "middle");  // Center the text horizontally
+                    textElement.setAttribute("alignment-baseline", "middle");  // Center the text vertically
+                    textElement.textContent = deonticValue.toUpperCase();  // Set Text and add uppercase
```

Comment:
Shall we keep the text formatting the same as for the text inside the nodes? It's not formatted differently in the example image, so I feel like this formatting would give it 'undeserved' importance.